ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/methods/PermJsonRpcMethodsTest.java

```diff
@@ -88,27 +77,4 @@ public void allowlistMethodsPresent() {
     assertThat(rpcMethods.get(PERM_REMOVE_NODES_FROM_ALLOWLIST.getMethodName()))
         .isInstanceOf(PermRemoveNodesFromAllowlist.class);
   }
-
-  @Deprecated
-  @Test
-  public void whitelistMethodsPresent() {
-    final Map<String, JsonRpcMethod> rpcMethods = permJsonRpcMethods.create();
-    assertThat(rpcMethods.size()).isEqualTo(13);
```

Comment:
This assert appears to be missing from allowlistMethodsPresent - if that matters?

Reply:
I don't think that's particularly useful so left it out